### PR TITLE
Fix up screen off timer of crkbd

### PR DESCRIPTION
**Pull request by comaid**
This patch fixes the following two problems related screen off timer on crkbd.

1. Never screen off because ScreenOffInterval exceeds uint16_t
2. Do not return from screen off until there is no change in the screen contents between screen off and on

### CODE REVIEW

**Comment by drashna** [comment]
Could you open a PR that does this for the Helix as well?  Or add it to this one?

**Comment by comaid** [comment]
Okey,I will. Sorry if I took time as I am a github beginner :-)

**Comment by comaid** [comment]
Sorry, I made a mistake and added it here once. I will open another PR.

**Comment by drashna** [comment]
Also, @foostan, this look good?

File: keyboards/crkbd/ssd1306.c [hunk 1]
```diff
@@ -321,12 +326,19 @@ void iota_gfx_task_user(void) {
 void iota_gfx_task(void) {
   iota_gfx_task_user();
 
-  if (display.dirty) {
+  if (display.dirty|| force_dirty) {
     iota_gfx_flush();
+        force_dirty = false;
```
Comment (foostan) [inline keyboards/crkbd/ssd1306.c]: Please fix the indent.

Comment (drashna) [reply keyboards/crkbd/ssd1306.c]:
```suggestion
    force_dirty = false;
```

Comment (comaid) [reply keyboards/crkbd/ssd1306.c]: I fixed it, thank you.

Comment (foostan) [reply keyboards/crkbd/ssd1306.c]: OH! Looks good `Suggested change`.

**Comment by foostan** [comment]
LGTM 👍

**Comment by drashna** [comment]
Thanks!

**Pull request merged**